packages/grid/x-data-grid/src/hooks/features/export/useGridPrintExport.tsx

```diff
@@ -158,9 +160,11 @@ export const useGridPrintExport = (
         gridFooterElementHeight
       }px`;
 
-      // Remove all loaded elements from the current host
-      printDoc.body.innerHTML = '';
-      printDoc.body.appendChild(gridClone);
+      // printDoc.body.appendChild(gridClone); should be enough but a clone isolation bug in Safari
```

Comment:
If you have a link to the bug report to help future evolution of this method

Reply:
I can't find it in https://bugs.webkit.org/buglist.cgi?quicksearch=cloneNode.

Reply:
On Safari 16.0, after a print without using `innerHTML` to cleanly deep clone:

<img width="780" alt="Screenshot 2022-09-26 at 13 14 07" src="https://user-images.githubusercontent.com/3165635/192262721-209ba09f-f3f2-4260-ac64-10d52ec66490.png">